resources/lib/ipwww_radio.py

```diff
@@ -578,26 +517,16 @@ def ParseStreams(stream_id):
     retlist = []
     # print "Parsing streams for PID: %s"%stream_id[0]
     # Open the page with the actual strem information and display the various available streams.
-    NEW_URL = "http://open.live.bbc.co.uk/mediaselector/5/select/version/2.0/mediaset/iptv-all/vpid/%s" % stream_id[0]
+    NEW_URL = "http://open.live.bbc.co.uk/mediaselector/5/select/version/2.0/mediaset/apple-ipad-hls/vpid/%s/proto/http?cb=%d" % (stream_id[0], random.randrange(10000,99999)) #NOTE magic from get_iplayer
```

Comment:
Why does this use a random magic? I feel quite uncomfortable with something like this, especially if it comes from get_iplayer. Don't get me wrong, they do great code, but I am under the impression that anything coming from get_iplayer is more likely to be broken by the Beeb.